Tests/Functional/Sources/ErrorHandlingTest.m

```diff
@@ -147,8 +147,7 @@ - (void)testRotationDescriptionGlossary {
 - (void)testKeyboardDismissalError {
   NSError *error;
   [EarlGrey dismissKeyboardWithError:&error];
-  NSString *keyboardErrorString = @"Failed to dismiss keyboard since it was not showing. Internal "
-                                  @"Error: Failed to dismiss keyboard as it was not shown.";
```

Comment:
Is this no longer showing?

Reply:
It's redundant, right? Why do we need to say the same error twice?